ormconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+    "type": "sqlite",
+    "host": "localhost",
+    "port": 3306,
+    "username": "sqlite",
+    "password": "sqlite",
+    "database": "mentoring_db",
+    "synchronize": true,
+    "logging": false,
+    "entities": [
+       "src/models/**/*.ts"
+    ],
+    "migrations": [
+       "src/migration/**/*.ts"
+    ],
+    "subscribers": [
+       "src/subscriber/**/*.ts"
+    ]
+ }
```

Comment:
@bigLucas 
Looking at PF project, I noticed that _ormconfig.json_ is autogenerated via _writeFile_, and its settings were elsewhere (_ConnectionsOptions.ts_).

Do you think should I keep as it is now or implement de PF structure?

Reply:
Hi @Aristevao,
We can keep our little project simple, you can just delete this file.